Sources/CustomDump/Conformances/SwiftUI.swift

```diff
@@ -22,6 +22,7 @@
           Mirror(reflecting: self).children.first?.value as Any,
           name: nil,
           indent: 2,
+          isRoot: true,
```

Comment:
This outlier on `Animation` is showing maybe a deficiency in how we're defining its dump...

Reply:
We could maybe add an optional requirement `customDumpDescription(isAbbreviated:)` to allow descriptions to differentiate...but maybe there's a better idea.